config/values.yaml.template

```diff
@@ -50,7 +48,7 @@ JWTPublicCert: <FILENAME>
 
 
 # Optionally edit the values below.
-
+Namespace: dss-main
```

Comment:
Let's add a warning that the certs scripts and some README commands will not work properly if this value is not "dss-main".